web/controllers/attendance_controller.ex

```diff
@@ -0,0 +1,23 @@
+defmodule CoursePlanner.AttendanceController do
```

Comment:
I was thinking of in the next pull request I can filter "fill" or "show" functionality based on the role.